package.json

```diff
@@ -61,7 +61,7 @@
     "lint:watch": "npm-run-all watch:ts -- -c \"npm-run-all lint\"",
     "prettier": " prettier-tslint fix \"packages/**/src/**/*.(ts|tsx)\"",
     "publish": "lerna publish",
-    "start": "TSC_WATCHFILE=UseFsEventsWithFallbackDynamicPolling npm-run-all -p build:watch start:examples",
+    "start": "cross-env TSC_WATCHFILE=UseFsEventsWithFallbackDynamicPolling npm-run-all -p build:watch start:examples",
```

Comment:
This is unrelated, just something that must have been added a while ago and it wasn't done properly (cross platform)

Reply:
perfect